swagger.js

```diff
@@ -16,10 +16,10 @@ const doc = {
     // Here we can define objects to use in our swagger documentation
     // These can be used as examples for request bodies, response bodies, etc
     PasswordlessUser: {
-      name: 'John Doe',
-      username: 'johndoe',
-      email: 'john@example.com',
-      role: { '@enum': ['user', 'admin'] },
+      $name: 'John Doe',
```

Comment:
Let's add the id field here as well just to fully document the expected response:

```suggestion
      $id: '64d6ca92a15d2e18ab96a2a3',
      $name: 'John Doe',
```